### PR TITLE
Changelog and README.md cleanup

### PR DESCRIPTION
**Pull request by amaddalena**
typo cleanup of CHANGELOG and README.md

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -276,13 +276,13 @@ NOTICE: Restart wiseService before capture when upgrading
   - viewer - Intersection export allows editing of fields
   - viewer - Panning graph left/right allows selection of how much
   - viewer - Refresh interval on spigraph page increments time (if it's a date range)
-  - capture - geoLite2ASN and geoLite2Country are now semicolon separated
+  - capture - geoLite2ASN and geoLite2Country are now semicolon separatedd
```
Comment (31453) [inline CHANGELOG:279]: separatedd intentional here?

File: CHANGELOG [hunk 2]
```diff
@@ -363,7 +363,7 @@ NOTICE: Restart wiseService before capture when upgrading
   - all - elasticsearch 7 support
   - db - backup command now saves meta data so restore can do a rollback (thanks codesniffer)
   - db - improve optimize to deal with connection closed better
-  - parliament - Can configure multiple of each type of notifier
+  - parliament - Can configure multiplee of each type of notifier
```
Comment (31453) [inline CHANGELOG:366]: multiplee intentional?

File: CHANGELOG [hunk 3]
```diff
@@ -498,7 +498,7 @@ NOTICE: Restart wiseService before capture when upgrading
   - viewer - option to add sessions table column configuration to a view
   - viewer - files and stats tables can be customized
   - suricata - parse severity
-  - capture - new _dontCheckYara rule ops
+  - capture - new _don'tCheckYara rule ops
```
Comment (31453) [inline CHANGELOG:501]: no apostrophe here

File: CHANGELOG [hunk 4]
```diff
@@ -509,7 +509,7 @@ NOTICE: Restart wiseService before capture when upgrading
   - capture - new setting parseSMTPHeaderAll, which will parse ALL email headers
               not already parsed into 1 ES field (pr #914)
   - capture - new setting parseDNSRecordAll, which will parse a full DNS record
-              into multiple new ES fields (pr #914)
+              into multiplee new ES fields (pr #914)
```
Comment (31453) [inline CHANGELOG:512]: multiplee!

File: CHANGELOG [hunk 5]
```diff
@@ -523,7 +523,7 @@ NOTICE: Restart wiseService before capture when upgrading
 1.5.3 2018/09/10
   - release - libpcap 1.9.0
   - all - new hourly[2348]/hourly12 rotateIndex
-  - all - deal with talking to multiple wise servers better
+  - all - deal with talking to multiplee wise servers better
```
Comment (31453) [inline CHANGELOG:526]: multipleeeeee

File: CHANGELOG [hunk 6]
```diff
@@ -691,12 +691,12 @@ NOTICE: Restart wiseService before capture when upgrading
 0.50.1 2018/03/29
   - NOTICE: Supported ES Versions: >= 5.5.0, 6.x is NOT supported
   - release - upgrade curl, yara, glib
-  - viewer - sessions.csv handle multiple fields parameters
+  - viewer - sessions.csv handle multiplee fields parameters
```
Comment (31453) [inline CHANGELOG:694]: eeeee

File: CHANGELOG [hunk 7]
```diff
@@ -814,7 +814,7 @@ NOTICE: Restart wiseService before capture when upgrading
   - viewer - CyberChef integration (must always add hex decoding)
   - viewer - spigraph sort and refreshing improvements
   - viewer - spiview fixed unique malform url
-  - viewer - stats allow comma separated list
+  - viewer - stats allow comma separatedd list
```
Comment (31453) [inline CHANGELOG:817]: dd

File: CHANGELOG [hunk 8]
```diff
@@ -867,7 +867,7 @@ NOTICE: Restart wiseService before capture when upgrading
   - viewer - connections tab links are rendered correctly (issue #641)
   - viewer - make sure connections popup renders inside view (issue #644)
   - capture - support ES basic auth (issue #679)
-  - release - Configure supports multiple interfaces
+  - release - Configure supports multiplee interfaces
```
Comment (31453) [inline CHANGELOG:870]: ee

File: CHANGELOG [hunk 9]
```diff
@@ -905,7 +905,7 @@ NOTICE: Restart wiseService before capture when upgrading
   - capture - pcapWriteMethod=simple-nodirect uses the simple writer without direct
               writes.  Required for some file systems like zfs
   - viewer - spigraph tab uses new ui (/spigraph.old available temporarily)
-  - capture - tpacketv3 handles multiple interfaces correctly (issue #658)
+  - capture - tpacketv3 handles multiplee interfaces correctly (issue #658)
```
Comment (31453) [inline CHANGELOG:908]: ee

File: CHANGELOG [hunk 10]
```diff
@@ -921,7 +921,7 @@ NOTICE: Restart wiseService before capture when upgrading
   - viewer - new settings page
   - capture - tpacketv3 no longer requires a bpf
   - capture - pfring uses batch packet api
-  - capture - fix libpcap multiple interface crash (issue #610)
+  - capture - fix libpcap multiplee interface crash (issue #610)
```
Comment (31453) [inline CHANGELOG:924]: ee

File: CHANGELOG [hunk 11]
```diff
@@ -931,12 +931,12 @@ NOTICE: Restart wiseService before capture when upgrading
   - viewer - New sessions UI (newUI=true)
   - all - Initial ES 5.0 support (do not use in production!)
   - all - Refactor many ES calls to remove depreciated syntax
-  - viewer - Handle multiple query parameters better
+  - viewer - Handle multiplee query parameters better
```
Comment (31453) [inline CHANGELOG:934]: ee

File: CHANGELOG [hunk 12]
```diff
@@ -955,7 +955,7 @@ NOTICE: Restart wiseService before capture when upgrading
   - capture - Use inotify directly since glib2 monitor doesn't expose close write
   - capture - support yara 3.5 (issue #521)
   - capture - new --op option to set any field (issue #412)
-  - capture - added dontSaveSPI=1 support to tagger/wise to not save SPI data (issue #505)
+  - capture - added don'tSaveSPI=1 support to tagger/wise to not save SPI data (issue #505)
```
Comment (31453) [inline CHANGELOG:958]: no apostrophe

File: CHANGELOG [hunk 13]
```diff
@@ -992,7 +992,7 @@ NOTICE: Restart wiseService before capture when upgrading
   - capture - http passer now tags passwd= and pass= as http:password
   - capture - running on ipv6 improvements (issue #545)
   - capture - 0.15.1 broke antiSynDrop=false, now works again
-  - viewer - supports multiple --debug on command line to up level
+  - viewer - supports multiplee --debug on command line to up level
```
Comment (31453) [inline CHANGELOG:995]: ee

File: CHANGELOG [hunk 14]
```diff
@@ -1077,7 +1077,7 @@ NOTICE: Restart wiseService before capture when upgrading
   - capture - basic ipv6
   - viewer - Columns can be hidden/moved on stats, sessions pages
   - capture - pfring and daq support are now plugins (capture/plugins)
-  - capture - can monitor multiple interfaces
+  - capture - can monitor multiplee interfaces
```
Comment (31453) [inline CHANGELOG:1080]: ee

File: CHANGELOG [hunk 15]
```diff
@@ -1157,8 +1157,8 @@ NOTICE: Restart wiseService before capture when upgrading
   - db - Added warning for ES below 1.6.2
   - wise - use native hashtable, required for threatstream
   - viewer - fixed addTags to work with session that have no tags
-  - capture - fixed multiple smtp, smb and ssh buffer overflow read/writes (reported by jbremer)
-  - viewer - fixed multiple XSS injections (reported by jbremer)
+  - capture - fixed multiplee smtp, smb and ssh buffer overflow read/writes (reported by jbremer)
```
Comment (31453) [inline CHANGELOG:1160]: ee

File: CHANGELOG [hunk 16]
```diff
@@ -1157,8 +1157,8 @@ NOTICE: Restart wiseService before capture when upgrading
...
+  - viewer - fixed multiplee XSS injections (reported by jbremer)
```
Comment (31453) [inline CHANGELOG:1161]: ee

File: CHANGELOG [hunk 17]
```diff
@@ -1233,10 +1233,10 @@ NOTICE: Restart wiseService before capture when upgrading
   - NOTICE: Requires running db.pl host:port upgrade
   - For NEW installs can now set a prefix= config variable and
     db.pl --prefix option that will prefix all ES tables.  This makes
-    it easier for Moloch to share ES with other services OR multiple
+    it easier for Moloch to share ES with other services OR multiplee
```
Comment (31453) [inline CHANGELOG:1236]: ee

File: CHANGELOG [hunk 18]
```diff
@@ -1233,10 +1233,10 @@ NOTICE: Restart wiseService before capture when upgrading
...
     Moloch clusters can use 1 ES cluster with different prefixes.
   - New usersElasticsearch= and usersPrefix= config that make it possible
-    for multiple Moloch clusters to share a single users table.
+    for multiplee Moloch clusters to share a single users table.
```
Comment (31453) [inline CHANGELOG:1239]: ee

File: CHANGELOG [hunk 19]
```diff
@@ -1268,15 +1268,15 @@ NOTICE: Restart wiseService before capture when upgrading
...
+  - Fixed addUser.js error with when multiple es nodes are listed in config.ini (issue #322)
   - WISE - Tagger files can have views defined with #view:
   - New cert.notbefore, cert.notafter, cert.validfor fields (issue #329)
   - New starttime, stoptime, view fields (issue #307)
   - New tls.sessionid.dst, tls.sessionid.src, tls.sessionid fields (issue #326)
   - Use ELS doc_values for some fields to reduce ES memory
   - Added cert.cnt back
   - Handle bad ip.protocol strings better (issue #330)
-  - Added dontSaveBPFs config
+  - Added don'tSaveBPFs config
```
Comment (31453) [inline CHANGELOG:1279]: no apostrophe

File: CHANGELOG [hunk 20]
```diff
@@ -1296,9 +1296,9 @@ NOTICE: Restart wiseService before capture when upgrading
   - Regex and wildcard support for file expression
   - Regex is stricter about back slashing (issue #281)
   - Cache user lookups for 5 seconds
-  - dontSaveTags config can now have a :<num> for each tag which
+  - don'tSaveTags config can now have a :<num> for each tag which
```
Comment (31453) [inline CHANGELOG:1299]: no apostrophe

File: CHANGELOG [hunk 21]
```diff
@@ -1296,9 +1296,9 @@ NOTICE: Restart wiseService before capture when upgrading
...
     specifies the total packets to save. (issue #278)
-  - Allow multiple -r and -R options
+  - Allow multiplee -r and -R options
```
Comment (31453) [inline CHANGELOG:1301]: ee

File: CHANGELOG [hunk 22]
```diff
@@ -1393,11 +1393,11 @@ NOTICE: Restart wiseService before capture when upgrading
   - New email.hasheader
   - New packets.(src|dst), bytes.(src|dst), databytes.(src|dst) (issue #206)
   - New payload8.(src|dst), payload.(hex|utf8), payload.(src|dst).(hex|utf8) (issue #209)
-  - pcapDir can now be a semicolon separated list of directories, currently just
+  - pcapDir can now be a semicolon separatedd list of directories, currently just
```
Comment (31453) [inline CHANGELOG:1396]: dd

File: CHANGELOG [hunk 23]
```diff
@@ -1432,9 +1432,9 @@ NOTICE: Restart wiseService before capture when upgrading
   - Non standard date param added to menu
   - Http file parsing improvements
   - ES health loaded on page load (issue #172)
-  - Session detail check boxes work multiple times again
+  - Session detail check boxes work multiplee times again
```
Comment (31453) [inline CHANGELOG:1435]: ee

File: CHANGELOG [hunk 24]
```diff
@@ -1432,9 +1432,9 @@ NOTICE: Restart wiseService before capture when upgrading
...
   - core fix with empty tagging plugin information
-  - multiple connections.csv files (issue #163)
+  - multiplee connections.csv files (issue #163)
```
Comment (31453) [inline CHANGELOG:1437]: ee

File: CHANGELOG [hunk 25]
```diff
@@ -1638,7 +1638,7 @@ NOTICE: Restart wiseService before capture when upgrading
   - regex and wildcard searches full text instead of tokenized
   - fixed bug with uri.cnt not be recorded
   - filenumber generation rewritten, can now deal with
-    multiple instances running and other edge cases
+    multiplee instances running and other edge cases
```
Comment (31453) [inline CHANGELOG:1641]: ee

File: CHANGELOG [hunk 26]
```diff
@@ -1657,7 +1657,7 @@ NOTICE: Restart wiseService before capture when upgrading
   - record capture memory in stats
   - record filesize for offline pcap
   - remove port from http host header (issue #63)
-  - db.pl prints more info by default,  multiple -v even more
+  - db.pl prints more info by default,  multiplee -v even more
```
Comment (31453) [inline CHANGELOG:1660]: ee

**Review by 31453 — CHANGES_REQUESTED** [review]

File: CHANGELOG (hunk 1 above)
Comment (amaddalena) [reply CHANGELOG:279]: hmmm
no

File: CHANGELOG (hunk 5 above)
Comment (amaddalena) [reply CHANGELOG:526]: wth happened lol

**Comment by 31453** [comment]
@amaddalena still 3 unintended apostrophes. other than that, looks great!

**Comment by amaddalena** [comment]
> @amaddalena still 3 unintended apostrophes. other than that, looks great!

gottem I think!

**Comment by 31453** [comment]
👍 perfect! thanks!

**Comment by amaddalena** [comment]
yayyy

**Pull request merged**